src/locales/en-US.po

```diff
@@ -0,0 +1,2750 @@
+msgid ""
```

Comment:
why all these changes?

Reply:
And its still not working ... Well isn't that file being auto-generated based on this command yarn i18n:compile, which should be running as part of postinstall but I'm not sure it is. So I've ran it locally and then committed that file.

Reply:
Because I'm getting this instead of variable value
![screencapture-pr1917-gpswapui-review-gnosisdev-2021-11-29-23_15_07](https://user-images.githubusercontent.com/34926005/143951372-06b15099-5dc0-462c-a682-aead83574dad.png)

Reply:
So looks like, either postinstall script is not running properly in the ci build process or the cache files are not updated with newer locale files. By adding `yarn i18n:compile` to build script in package.json fixed the issue.